### ISSUE TITLE
Google playstore downloads issue

### ISSUE THREAD

**Issue by drixtol** [uklans/cache-domains#200]
### Describe the issue you are having
google play store apps fail to download (android 12.x)

### Describe your setup?
lancache monolithic to pihole to cloudflare

with the new [google ](https://github.com/uklans/cache-domains/blob/master/google.txt) file the store cant download. Everything else seems to be working fine. The apps will go in to pending and then fail after 30 seconds. 

I forked the repo and removed google, pointed my cache repo to that and it worked with no other changes. Changed it back and it stopped working.

**Comment by VibroAxe** [comment]
@drixtol I have temporarily reverted the PR for google, I'd be interested to know if this resolves your problem?

**Comment by drixtol** [comment]
yup, restarted to get the refresh and it immediately started working.

**Comment by VibroAxe** [comment]
Can you clarify what actually stopped. We talking the play store on a phone, chromebook store, chrome web store or something else?

**Comment by drixtol** [comment]
Android 11 (samsung S7 tablet) and 12(pixel 6) both fail. Interestingly enough, android 10(samsung S9) works.
![Screenshot_20220818-144624](https://user-images.githubusercontent.com/20601906/185501280-add990c1-6936-4c0e-bcb8-f296dc8958f4.png)
![Screenshot_20220818-144755](https://user-images.githubusercontent.com/20601906/185501288-24cd56d9-fd6c-46d7-b215-1269a92ec7b5.png)

**Comment by VibroAxe** [comment]
Out of interest, are you running sniproxy or monlithic for the 443 ssl port?

**Comment by drixtol** [comment]
monolithic
